core/src/main/resources/hudson/triggers/SlowTriggerAdminMonitor/description_fr.properties

```
@@ -0,0 +1 @@
+blurb = Ce message avertit les administrateurs que les déclencheurs ne se déclenchent pas comme planifié, typiquement car des plugins exécutent des actions additionnelles (comme des accès réseau) durant l''exécution des déclencheurs.
```

```suggestion
blurb = Ce message avertit les administrateurs que les déclencheurs ne suivent pas le planning, typiquement car des plugins exécutent des actions additionnelles (comme des accès réseau) durant l''exécution des déclencheurs.
```
Not an easy one ^^ Optional suggestion.